test/data/dos.md

```diff
@@ -0,0 +1,2549 @@
+Without additional checks, this file causes a segfault in PHP < 7.0, which is caused by Regex Backtrack*
```

Comment:
Could we maybe generate these results at runtime in some dedicated tests, as opposed to adding 10k lines to the repo? :p

Reply:
given that all tests work in the same way by reading files imo its the most straight forward way to just have these files here. isn't disk space is cheap enough? :)